modules/fame_azure-vpn/filters.tf

```diff
@@ -0,0 +1,5 @@
+locals {
```

Comment:
The filters should be a link to https://github.com/claranet/terraform-signalfx-detectors/blob/master/common/module/filters-integration-azure.tf

Reply:
As discussed internally, I've reverted the link to filters-integration-azure. FAME act as an agent so it must use non prefixed tags.